CLOUD-61854 Fix ssh description in the documentation

akanto opened the pull request:
@dbialek2 please take a look

dbialek2 commented:
@akanto 
Can you change this a little bit to something like:

To ssh to a running VM, you need to know its public IP address and private key. You can find the IP addresses of all the running VMs in the Cloudbreak UI, on the Cluster details page, in the *Nodes* section. Only key-based authentication is supported. The private key that you need to use to access the VM is the counterpart of the public key that you specified when creating a cloud credential.

You can use the `cloudbreak` user to SSH to the VM. This user has passwordless sudo rights. 

For example:
```
ssh -i ~/.ssh/your-private-key.pem cloudbreak@<public-ip>
```

dbialek2 commented:
@akanto I noticed that under "Ambari Database", it says:

"To access Ambari's database, SSH to the selected node and run the following command:"

But there is no command, just a new Proxy Settings section. Is there a command missing there?

akanto commented:
@dbialek2 Thanks, updated as suggested

Also, removed some part of the documentation including what you noticed, since that part is not relevant anymore, because we removed docker from the cluster.
